feat: global secret search (#173)
phasehq/docs: src/pages/console/secrets.mdx

![view secret history](/assets/images/console/secrets/secret-history-restore.png)

## Search for a Secret

You can search for a secret by typing the secret key into the search bar in a single environment or across all environments in the App Secrets tab. This will filter the list of secrets to only show secrets that match the search query.

### Global Secret search

> You can also search for secrets across all Apps, Environments, and Folders in your organization that you have access to by triggering the command palette and typing secret key. You will see matching results listed with the App, Environment, and Path context of each secret.

Reviewer: ```suggestion
You can also search for secrets across all Apps, Environments, and Folders that you have access to in your organization via the command palette. You will see matching results listed with the App, Environment, and Path context of each secret.
```

<Note>

> All secret searches are performed client side on in your browser on encrypted data regardless of App encryption mode. (End-to-end or Server-side Encryption)

Reviewer: ```suggestion
All secret searches are performed client side in your browser regardless of App encryption mode. Secret values are not fetched or decrypted when performing searches.
```
